src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -0,0 +1,94 @@
+<Project Sdk="Microsoft.NET.Sdk.IL">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0;net7.0</TargetFrameworks>
```

Comment:
Should I keep net7.0 now that it's out of support?

Reply:
`net7.0` doesn't get compiled, it's just a "placeholder" TFM to create a placeholder file. So it should be kept.

Reply:
The placeholder condition applies to all .NET Core App versions:
```xml
<IsPlaceholderTargetFramework Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</IsPlaceholderTargetFramework>
```

Reply:
But as you said in another comment, we don't compile against netcoreapp, so I removed the preprocessor branches that applied to it.

I mainly ask since you left a question if we would have a bad failure when consuming this in .NET 6.0.

Reply:
> The placeholder condition applies to all .NET Core App versions:

Yes but it was net7.0 when we moved the API inbox so the placeholder needs to be under the "lib/net7.0" folder in the package.